app/src/main/res/values-nn/strings.xml

```diff
@@ -358,4 +373,130 @@
+  <string name="app_settings_blur_nsfw_except_from_nsfw_communities">NSFW utanom frå NSFW-fellesskap</string>
+  <string name="app_settings_blur_nsfw_except_from_nsfw_inside_community">Utanom NSFW inni fellesskap</string>
+  <string name="time_ago_failed_to_parse">Klarte ikkje tolka dato/klokkeslett</string>
+  <string name="comment_gone">Det finst inga registrering av kommentaren</string>
+  <string name="server_version_not_supported">Versjonen (%s) er ikkje støtta</string>
+  <string name="comment_removed">Kommentar fjerna</string>
+  <string name="comment_restored">Kommentar gjenoppretta</string>
+  <string name="post_removed">Innlegg fjerna</string>
+  <string name="post_restored">Innlegg gjenoppretta</string>
+  <string name="remove_comment">Fjern kommentar</string>
+  <string name="restore_comment">Gjenopprett kommentar</string>
+  <string name="remove_post">Fjern innlegg</string>
+  <string name="restore_post">Gjenopprett innlegg</string>
+  <string name="lock_post">Lås innlegg</string>
+  <string name="unlock_post">Lås opp innlegg</string>
+  <string name="type_your_reason">Skriv inn grunngjeving</string>
+  <string name="swipe_action_preset_disabled">Slått av</string>
+  <string name="swipe_action_preset_default">To sider</string>
+  <string name="swipe_action_preset_downvote_on_left_upvote_on_right">Venstre/høgre røyster</string>
+  <string name="only_right_swipe_action_preset">Berre sveip til høgre</string>
+  <string name="only_left_swipe_action_preset">Berre sveip til venstre</string>
+  <string name="person_banned">Utestengde %1$s</string>
+  <string name="person_banned_for_x_days">Utestengde %1$s i %2$d dagar</string>
+  <string name="person_banned_from_community">Utestengde %1$s frå %2$s</string>
+  <string name="person_banned_from_community_for_x_days">Utestengde %1$s frå %2$s i %3$d dagar</string>
+  <string name="person_unbanned">Fjerna utestenging av %1$s</string>
+  <string name="person_unbanned_from_community">Fjerna utestenging av %1$s frå %2$s</string>
+  <string name="ban_person">Utesteng %1$s</string>
+  <string name="ban_person_from_community">Utesteng %1$s frå %2$s</string>
+  <string name="unban_person">Fjern utestenging av %1$s</string>
+  <string name="unban_person_from_community">Fjern utestenging av %1$s frå %2$s</string>
+  <string name="remove_content">Fjern innhald</string>
+  <string name="permaban">Utesteng for godt</string>
+  <string name="days_until_expiration">Utgår (dagar)</string>
+  <string name="feature_in_community">Framhev i fellesskapet</string>
+  <string name="unfeature_in_community">Fjern framheving frå fellesskapet</string>
+  <string name="feature_in_local">Framhev lokalt</string>
+  <string name="unfeature_in_local">Fjern framheving lokalt</string>
+  <string name="swipe_to_action_presets">Innstilling for sveipe­rørsle</string>
+  <string name="swipe_action_preset_only_votes">Berre røyster</string>
+  <string name="swipe_action_preset_only_votes_mirrored">Only votes (spegla)</string>
```

Comment:
Here too I believe

Reply:
You’re right. Thanks for the thorough review! Now fixed.